src/Tizen.Network.Smartcard/Tizen.Network.Smartcard/SmartcardManagerImpl.cs

```diff
@@ -98,6 +92,7 @@ internal IEnumerable<SmartcardReader> GetReaders()
         {
             IntPtr readerPtr;
             int len = 0;
+            List<SmartcardReader> readerList = new List<SmartcardReader>();
 
             int ret = Interop.Smartcard.GetReaders(out readerPtr, out len);
```

Comment:
It seem to memory leak, if not release readerPtr, 

```
142/**
143 * @brief Gets the list of available Secure Element readers.
144 * @since_tizen 2.3.1
145 * @ingroup CAPI_NETWORK_SMARTCARD_SE_SERVICE_MODULE
146 *
147 * @remarks readers must be released using free().
148 *
149 * @param[out] readers List of readers
150 * @param[out] length The number of readers
151 *
152 * @return 0 on success, otherwise a negative error value.
153 * @retval #SMARTCARD_ERROR_NONE Successful
154 * @retval #SMARTCARD_ERROR_NOT_SUPPORTED Not supported
155 * @retval #SMARTCARD_ERROR_INVALID_PARAMETER Invalid function parameter
156 * @retval #SMARTCARD_ERROR_GENERAL A general error occurred
157 * @retval #SMARTCARD_ERROR_NOT_INITIALIZED Smartcard service not initialized
158 *
159 */
```

Reply:
I add the free method for readerPtr